uberduck_ml_dev/trainer/tacotron2.py

```diff
@@ -550,25 +559,58 @@ def validate(self, **kwargs):
 
     @property
     def val_dataset_args(self):
-        val_args = [a for a in self.training_dataset_args]
-        val_args[0] = self.val_audiopaths_and_text
-        return val_args
+        #         val_args = {}
+        #         val_args = [a for a in self.training_dataset_args]
+        #         val_args[0] = self.val_audiopaths_and_text
+        #         return val_args
+        return {
+            "audiopaths_and_text": self.val_audiopaths_and_text,
+            "text_cleaners": self.text_cleaners,
+            "p_arpabet": self.p_arpabet,
+            "n_mel_channels": self.n_mel_channels,
+            "sampling_rate": self.sampling_rate,
+            "mel_fmin": self.mel_fmin,
+            "mel_fmax": self.mel_fmax,
+            "filter_length": self.filter_length,
+            "hop_length": self.hop_length,
+            "win_length": self.win_length,
+            "symbol_set": self.symbol_set,
+            "max_wav_value": self.max_wav_value,
+            "pos_weight": self.pos_weight,
+        }
```

Comment:
to avoid duplicating code and avoid errors, would it be useful to use a similar pattern as before in terms of defining `self.val_dataset_args` in terms of `self.training_dataset_args`?

For example:
```python
args = dict(**self.training_dataset_args)
args["audiopaths_and_text"] = self.val_audiopaths_and_text
return args
```